CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # activerecord-postgres_pub_sub
 
+## v2.0.0 (unreleased)
```

Comment:
cut a new major version since we're dropping a version support for `pg` but happy to change if folks feel differently